### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by jtrussell**
Adding GA in production.

### CODE REVIEW

File: next/next.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  env: {
+    GA_TRACKING_ID: 'UA-142178156-1',
```
Comment (alexmiddeleer) [inline next/next.config.js]: Is there a GA account with this tracking_id?

Comment (jtrussell) [reply next/next.config.js]: I assumed so https://trello.com/c/kTcQbXDh, perhaps this was a placeholder?

Comment (jtrussell) [reply next/next.config.js]: Not particularly sensitive but would you rather have externally managed secrets? That would be cool, probably out scope for this task.

Comment (alexmiddeleer) [reply next/next.config.js]: Yep totally

Comment (jtrussell) [reply next/next.config.js]: Right on - I'll be adding secrets as part of auth. Once that's in place we can move our GA ID there.

**Pull request merged**